### ISSUE TITLE
Copy/paste Oxygen component from one page/template to another

### ISSUE THREAD

**Issue by SinjoriS** [soflyy/oxygen-bugs-and-features#2212]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*
Oxygen lacks of some very basic features, such as copy-pasting components from one page/template to another. I am more and more aware of that issue, and that's something very handy in Divi (for instance). That increase the speed of building pages/templates in many cases. Ok, we can use Reusable blocks, but that's not the same thing. If I compare that feature in Divi and (jus plain copy-paste of the component), with Oxygen reusable blocks - Oxygen reusable blocks are miles away in terms of practicality. I am wasting so much time on that. It's just plain copy-paste function. Even plain Windows Note has it. 

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*
Any Website. It's plain simple and very basic feature. 

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*
Of course. Divi theme/builder. It works perfect there. I think Avada Fusion builder also has that (but I am not 100% sure), and some other page/theme builders. 
Oxygen is far more complex and allows us to so complex things, while it lacks of some basic features, like copy-paste feature.

**Comment by KittenCodes** [comment]
> Oxygen lacks of some very basic features, such as copy-pasting components from one page/template to another.

This is possible using the (third-party, paid) add-on, Hydrogen Pack: https://www.cleanplugins.com/products/hydrogen-pack/.

**Comment by SinjoriS** [comment]
> This is possible using the (third-party, paid) add-on, Hydrogen Pack: https://www.cleanplugins.com/products/hydrogen-pack/.

I know that. But, don't you think such BASIC feature should be implemented into Oxygen?
Or is is better to pay 40 USD / Per Site (or 70 USD for Unlimited Site) for that small think, which is implemented EVERYWHERE. From very simple text/code editors, to other page/theme builders. If you every try to work in Divi, for instance, you would see how this basic thing is handy. I miss more that simple thing, then any other complex features implemented into Oxygen.

**Comment by lothisoft** [comment]
I don't mind paying for add-ons if they are well built.  I think the thriving market for add-ons is a plus for Oxygen Builder.

**Comment by SinjoriS** [comment]
> I don't mind paying for add-ons if they are well built. I think the thriving market for add-ons is a plus for Oxygen Builder.

Lothar, you miss my point. I also pay for plugins, and I don't mind that, if they are well built. But, as I said - it's not the point if we wish to pay for addon or not, if it is well build and if it is worth it. 
The point is - Oxygen is missing a simple basic feature. That's a point. And, that's why I suggest that feature as "enhancement" here.

**Comment by elmichalos** [comment]
+1

**Comment by elmichalos** [comment]
Yeah, this is too basic not to be implemented. Why don't you strip other things from Oxy and make us pay for that? Because there is something out there it doesn't mean you cannot implement that into the core. And then I realize that I need to install 20 plugins to start working on the website. We are not asking you to implement motion.page functionality. Just simple copy/paste. Please
